scripts/import_libcbor.py

```diff
@@ -0,0 +1,139 @@
+import os
+import tempfile
+import shutil
+import subprocess
+import argparse
+
+def parse_version(version_string):
+    if version_string.startswith('v'):
+        version_string = version_string[1:]
+    parts = version_string.split('.')
+    if not parts:
+        raise ValueError("Invalid version string")
+    major = int(parts[0])
+    minor = 0
+    patch = 0
+    if len(parts) > 1:
+        minor = int(parts[1])
+
+    if len(parts) > 2:
+        patch = int(parts[2])
+
+    return major, minor, patch
```

Comment:
trivial: this can be simpler
```suggestion
    match = re.fullmatch(r'v(\d+)\.(\d+)\.(\d+)', version_string)
    if not match:
        raise ValueError("Invalid version string")
    return match.group(1), match.group(2), match.group(3)
```